modules/oxide/src/less/theme/components/suggestededits/suggestededits.less

```
@@ -123,248 +123,257 @@
       }
 
       .tox-suggestededits__sidebar-header {
-        padding: 4px 12px;
+        padding: 4px 0px;
       }
 
       .tox-suggestededits__sidebar-content {
         display: flex;
         flex: 1;
         flex-direction: column;
-        gap: 12px;
         overflow: scroll;
-        padding: 4px 12px;
+        overflow-y: auto;
+        overflow-x: auto;
```

`overflow` is shorthand for the `-x` and `-y` values
```suggestion
        overflow: auto;
```

---

Great point!